src/options.h

```diff
@@ -157,31 +158,33 @@ class AppOptions {
 struct HlsOptions final : public AppOptions {
 
   HlsOptions(AppOptions &parent):
-      AppOptions(parent, HLS_CMD, "High-level synthesis") {
-
+      AppOptions(parent, HLS_CMD, "High-level synthesis"),
+      // Initialize the output paths vector for every possible format.
+      outNames(OUTPUT_FORMATS_COUNT) {
+      
     // Named options.
     options->add_option(CONFIG_ARG, latConfigFile, "JSON latency configuration path")
-           ->expected(1);
+        ->expected(1);
     
     auto schedGroup = options->add_option_group(SCHEDULER_GROUP);
     schedGroup->add_flag(ASAP_SCHEDULER_FLAG, asapScheduler, "Use greedy as-soon-as-possible scheduler");
     schedGroup->add_flag(LP_SCHEDULER_FLAG,   lpScheduler,   "Use Linear Programming scheduler");
     schedGroup->require_option(1); 
-
-    options->add_option(OUT_ARG, outFile, "Output file path (default: standard output stream)")
-           ->expected(0, 1);
+    auto outputGroup = options->add_option_group(OUTPUT_GROUP);
+    outputGroup->add_option(SV_OUT_ARG, outNames[SV_OUT_ID], "File path for SystemVerilog output");
```

Comment:
"File path for SystemVerilog output" -> "Path to output SystemVerilog module"

Reply:
Fixed.